source/extensions/extensions_build_config.bzl

```diff
@@ -104,6 +104,12 @@ EXTENSIONS = {
 
     "envoy.filters.udp_listener.udp_proxy":             "//source/extensions/filters/udp/udp_proxy:config",
 
+    #
+    # Request ID utilities
+    #
+    
+    "envoy.request_id_utils.uuid":                      "//source/extensions/request_id_utils/uuid:config",
```

Comment:
Should the existing functionality be always linked with possible overriding? This will avoid build changes for all consumers that might rely on this functionality?